.github/workflows/houdini.yml

```diff
@@ -2,73 +2,62 @@
 name: Houdini
 
 on:
+  push:
+    branches:
+      - '**'
+    paths-ignore:
+      - 'CHANGES'
+      - 'openvdb/doc/**'
+      - 'openvdb_maya/**'
+      - 'openvdb_ax/**'
+      - '**/*.md'
+  pull_request:
+    branches:
+      - '**'
+    paths-ignore:
+      - 'CHANGES'
+      - 'openvdb/doc/**'
+      - 'openvdb_maya/**'
+      - 'openvdb_ax/**'
+      - '**/*.md'
```

Comment:
Can we move this back into the build.yml job? I would prefer all the build jobs to remain under the same heading. I think the original reason for wanting to split this was due to having different triggers, but it looks like they're the same now?

Reply:
This also allows for individual status badges (see here https://github.com/dneg/openvdb). tbh I prefer it this way, it's much clearer and the jobs are completely separate. The Houdini one may also change with the introduction of the AX SOP. Any of this worth it to you?

Reply:
IMO, those status badges are not particularly useful, maybe as a small motivation to us to fix CI issues in a timely fashion? As a user, I think it's fine to know if "something" is broken.

I'd like to optimise this for the developer experience and there are two issues that this implementation exposes in the GitHub Actions UI:

* if you group under multiple workflows, when you view one "run", you cannot easily jump to another that was triggered from the same commit, you have to go back to "actions" and then find the next from the list of all actions. Here's an example:

https://github.com/AcademySoftwareFoundation/openvdb/actions/runs/361703656

Now try and find the "houdini" run that was triggered from the same 184151f commit?

It's fine if you go via the checks page from the PR such as here:

https://github.com/AcademySoftwareFoundation/openvdb/pull/818/checks

But that means you have to navigate to the PR first and then to this page rather than go directly via actions.

* your automated matrix naming truncates the name of the build. From the same link as above - can you find which one is ABI=5 without having to hover over the name? I don't mind adopting this matrix approach, but I would at least like to fix the naming.

Reply:
I see how the first issue could catch somebody out, though I think it's a bit pedantic... I'll move it back for the sake of not stalling this PR

> your automated matrix naming truncates the name of the build. 

This has nothing to do with moving it, but I'll fix the naming

Reply:
Thanks. Agreed, it definitely reads as pedantic when viewed in light of this massive PR. :) I think in future we should probably make this kind of change in a separate PR. Many of these CI changes are unrelated to AX and could be reviewed in isolation.

Yes, the naming could have been a separate comment.

Reply:
I've fixed both issues, though when I took a look at the houdini naming it seems the old naming was even worse:

https://github.com/AcademySoftwareFoundation/openvdb/actions/runs/362887233
vs
https://github.com/AcademySoftwareFoundation/openvdb/actions/runs/361703653

"testhou175" vs " linux-vfx-hou:17_5-vfx:2018-cxx:cl..." (where the ... is the truncation)

So I'm not sure what you were referring too.. I'm removed the " linux-vfx-" prefix anyway

Reply:
Just realised I think you were referring to the linux build names? I've thus moved the abi number to the front

Reply:
Great, thank you!